cmd/kubelet/app/options/options.go

```diff
@@ -60,74 +51,16 @@ type KubeletServer struct {
 
 // NewKubeletServer will create a new KubeletServer with default values.
 func NewKubeletServer() *KubeletServer {
+	config := componentconfig.KubeletConfiguration{}
+	api.Scheme.Convert(&v1alpha1.KubeletConfiguration{}, &config)
 	return &KubeletServer{
 		AuthPath:              util.NewStringFlag("/var/lib/kubelet/kubernetes_auth"), // deprecated
 		KubeConfig:            util.NewStringFlag("/var/lib/kubelet/kubeconfig"),
 		DockerDaemonContainer: "/docker-daemon",
 
-		SystemReserved: make(util.ConfigurationMap),
-		KubeReserved:   make(util.ConfigurationMap),
-		KubeletConfiguration: componentconfig.KubeletConfiguration{
-			Address:                     "0.0.0.0",
-			CAdvisorPort:                4194,
-			VolumeStatsAggPeriod:        unversioned.Duration{time.Minute},
-			CertDirectory:               "/var/run/kubernetes",
-			CgroupRoot:                  "",
-			ConfigureCBR0:               false,
-			ContainerRuntime:            "docker",
```

Comment:
where did all the defaults get moved?

looking at the generated docs they are lost... for example, container runtime in the generated docs shows "" when it was "docker"